tests/Checks.hs

```diff
@@ -2,6 +2,7 @@
 module Main where
```

Comment:
Is this deliberately exporting testAdditive? Otherwise `module Main(main) where` is probably desirable. I tend to always require export lists.